addons/account/views/account_view.xml

```diff
@@ -1440,56 +1440,58 @@ action = model.setting_init_bank_account_action()
             <field name="model">account.tax.template</field>
             <field name="arch" type="xml">
                 <form string="Account Tax Template">
-                    <group>
```

Comment:
Could you do the same but with minimal diff? just putting the `<sheet>` tag with same alignment as the `<form>` tag

Reply:
I kinda wanted to indent / format on purpose hehe. It is master after all so I hope the diff will be accepted.